firmware/spade/README.md

```diff
@@ -83,6 +86,44 @@ cmake --build --preset=rpi
 
 A UF2 file will be outputted to `rpi_build/src/spade.uf2`. On macOS, with a Pico plugged in and in BOOTSEL mode, you can transfer from the CLI with `cp ./rpi_build/src/spade.uf2 /Volumes/RPI-RP2`.
 
+#### Debugging on the Pico with a probe
```

Comment:
I think this needs work - we want this to be MUCH MUCH easier for people.  We do not want people, at least initially (or perhaps even at all, unless they are very advanced users) to be doing many of these things manually - we should hide all of this stuff from the user, and instead provide them simple-to-use tools to execute the proper openocd/cmake/yadayada cmds.  This is one of the reasons we are investing in this gardenshed thing - if we properly support it and invest in it, the instructions can basically be ultra-concise - i.e., "run gardenshed help"...

Reply:
definitely! I'll remove this change from this PR for now since it's not specificially with SD support and the instructions are out of date, with the wonderful work on gardenshed and the possibility of debug shims to make wiring easier.